src/App.js

```diff
@@ -1,28 +1,69 @@
 import { useEffect, useState } from "react"
 import { Link, Route, Routes } from "react-router-dom"
+import { MutatingDots } from 'react-loader-spinner'
```

Comment:
I didn't know you could do this until your support ticket on Friday! So thank you for that. Such a nice solution compared to using an image and making it spin with css (which is what I originally did).